test/cql-pytest/test_tombstone_limit.py

```diff
@@ -247,8 +247,9 @@ def check_pages_many_partitions(results, expected):
         assert p == expected_pages.get(i, [])
 
 
-# xfail_tablets due to https://github.com/scylladb/scylladb/issues/16486
-def test_partition_tombstone_prefix(cql, test_keyspace, lowered_tombstone_limit, driver_bug_1, xfail_tablets):
+# xfail with tablets due to https://github.com/scylladb/scylladb/issues/16486
+@pytest.mark.parametrize("test_keyspace", [pytest.param("tablets", marks=[pytest.mark.xfail]), "vnodes"], indirect=True)
```

Comment:
Again, "reason" for xfail mark here and in the rest of this file

Reply:
Fixed in v2.